recipes/vectorscan/all/conanfile.py

```diff
@@ -0,0 +1,167 @@
+
+    def validate_build(self):
+        if self.settings.os == "Macos" and cross_building(self):
+            raise ConanInvalidConfiguration("Cross-building is not supported on macOS")
+
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.18.4 <4]")
+        self.tool_requires("ragel/6.10")
+        if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
+            self.tool_requires("pkgconf/[>=2.2 <3]")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+        apply_conandata_patches(self)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["BUILD_STATIC_LIBS"] = not self.options.shared
+        tc.variables["DEBUG_OUTPUT"] = self.options.debug_output
+        if self.options.dump_support != "auto":
+            tc.variables["DUMP_SUPPORT"] = self.options.dump_support
+        if "with_cpu_native" in self.options:
+            tc.variables["WITH_CPU_NATIVE"] = self.options.with_cpu_native
+        if "with_fat_runtime" in self.options:
+            tc.variables["WITH_FAT_RUNTIME"] = self.options.with_fat_runtime
+        if "with_avx" in self.options:
+            if not self.options.with_avx:
+                tc.cache_variables["BUILD_AVX2"] = False
+            elif self.options.with_avx == "avx2":
+                tc.cache_variables["BUILD_AVX2"] = True
+            elif self.options.with_avx == "avx512":
+                tc.variables["BUILD_AVX512"] = True
+            elif self.options.with_avx == "avx512vbmi":
+                tc.variables["BUILD_AVX512VBMI"] = True
+        if "with_sve" in self.options:
+            if not self.options.with_sve:
+                tc.cache_variables["BUILD_SVE"] = False
+            elif self.options.with_sve == "sve":
+                tc.cache_variables["BUILD_SVE"] = True
+            elif self.options.with_sve == "sve2":
+                tc.cache_variables["BUILD_SVE2"] = True
+            elif self.options.with_sve == "sve2_bitperm":
+                tc.cache_variables["BUILD_SVE2_BITPERM"] = True
+        tc.cache_variables["BUILD_CHIMERA"] = self.options.with_chimera
+        tc.variables["BUILD_EXAMPLES"] = False
+        tc.variables["BUILD_BENCHMARKS"] = False
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="LICENSE", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        cmake = CMake(self)
+        cmake.install()
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+
+    def package_info(self):
+        hs_include_dir = os.path.join("include", "hs")
```

Comment:
Both the generated pkgconfig pc file (With `Cflags: -I${includedir}/hs`) and the CMake examples in upstream expect the headers to be available directly